tests/mpi/petsc_step-27.cc

```diff
@@ -329,24 +329,24 @@ namespace Step27
     LA::MPI::Vector completely_distributed_solution(locally_owned_dofs,
                                                     mpi_communicator);
 
-    SolverControl solver_control(system_rhs.size(),
-                                 1e-8 * system_rhs.l2_norm());
-    //                           ^~~~
-    // Loss of precision with a factor of 1e-12 with Trilinos
+    SolverControl solver_control(
+      system_rhs.size(),
+      1e-8 * system_rhs.l2_norm(),
+      // ^~~~
+      // Loss of precision with a factor of 1e-12 with Trilinos
+      false,
+      false);
     LA::SolverCG cg(solver_control, mpi_communicator);
 
     LA::MPI::PreconditionAMG                 preconditioner;
     LA::MPI::PreconditionAMG::AdditionalData data;
     data.symmetric_operator = true;
     preconditioner.initialize(system_matrix, data);
 
-    check_solver_within_range(cg.solve(system_matrix,
-                                       completely_distributed_solution,
-                                       system_rhs,
-                                       preconditioner),
-                              solver_control.last_step(),
-                              5,
-                              40);
+    cg.solve(system_matrix,
```

Comment:
why this change?

Reply:
Since the test copies stdout/stderr to `output` (i.e., things printed to deallog were ignored) that check never made it to the stored `.output` files. I deleted it so that the test had the same output (aside from blank lines) as it did before.